src/components/BottomBar/BottomBarCourseReview.vue

```diff
@@ -119,17 +119,14 @@ export default defineComponent({
       return `https://www.cureviews.org/course/${subject}/${number}`;
     },
     CUROverallRating(): string | number {
-      if (this.courseObj.overallRating === 0) return '';
```

Comment:
No idea. Might be some guarding code in case they use 0 as placeholder. Better confirm with CUReviews?

Reply:
According to Akash they return `null` if there's no data.

I think either way there shouldn't be an instance where we use `''` instead of `N/A`, so it's probably fine to remove the code.